### PR TITLE
Contributing Guide

### LINKED ISSUE #1110: Add a Contributor Guide
We should document some of our contributor guidelines (pre-commit, tests, etc) now that we have them.

A good one to model after: https://github.com/microsoft/DeepSpeed/blob/master/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jahatef**
fixes #1110

### CODE REVIEW

File: configs/neox_arguments.md [hunk 1]
```diff
@@ -261,6 +261,14 @@ Model Arguments
 
 
 
+- **layernorm_fusion**: bool
```
Comment (Quentin-Anthony) [inline configs/neox_arguments.md:264]: why is this here?

Comment (jahatef) [reply configs/neox_arguments.md:264]: outstanding question. Will remove.

Comment (jahatef) [reply configs/neox_arguments.md:264]: Actually this belongs here. Just not in this commit. This is autogenerated by https://github.com/EleutherAI/gpt-neox/pull/1138/commits/e840a061611507f972f0d381eeaac48cc336b338. I think this is a results of our docs updater not working before

**Review by Quentin-Anthony — APPROVED** [review]
